client/components/leftPage.js

```diff
@@ -0,0 +1,9 @@
+import React, { PropTypes, Component } from 'react';
+
+var LeftPage = (props) => (
+  <div className="leftPage"> 
```

Comment:
This is being ultra picky, usually I think for html class names the naming is done as "left-page" rather than "leftPage" to distinguish it as a html name rather than JavaScript variable